config/clients/js/template/common.mustache

```diff
@@ -115,17 +116,13 @@ export const createRequestFunction = function (axiosArgs: RequestArgs, globalAxi
     }
     return async (axios: AxiosStatic = globalAxios) : PromiseResult<any> => {
         await setBearerAuthToObject(axiosArgs.options.headers, credentials!);
-        for (let i = 0; i < maxRetry + 1; i++) {
```

Comment:
Can we avoid doing a linear backoff in favor of an exponential backoff? My concern is that for clients issuing larger BatchChecks, it's possible to receive a large amount of load from multiple clients at the same time (e.g. thundering herd idea..).

Reply:
Look at: https://github.com/openfga/js-sdk/blob/da801b39a0026fae2af14afa4f7234579e9caa64/common.ts#L110